docs/core/testing/mstest-analyzers/mstest0007.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "MSTEST0007: Ensure test attributes can only be set on methods marked with TestMethod attribute"
```

Comment:
We usually put the rule title so it would be

```suggestion
title: "MSTEST0007: [<TestAttribute>] can only be set on methods marked with [TestMethod]"
```